CONTRIBUTING.md

```diff
@@ -3,6 +3,10 @@ We welcome all contributions to garage.
 
 Use this guide to prepare your contribution.
 
+# Post-setup instructions
+After activating your conda environment (e.g. `conda activate garage`), make sure you run `pre-commit install` to install the pre-commit package on your machine.
+This will streamline your git process and reduce potential CI errors.
```

Comment:
This will help streamline the pull request process by catching basic problems locally before they are checked by the CI.